--- NVAPIWrapper.cpp	1969-12-31 17:00:00.000000000 -0700
+++ RTSSOverlayEditor_NVAPIWrapper_modified.cpp	1969-12-31 17:00:00.000000000 -0700
@@ -7,7 +7,7 @@
 
 #include "Log.h"
 //////////////////////////////////////////////////////////////////////
-#pragma comment(lib, "nvapi.lib") 
+#pragma comment(lib, "../../thirdparty/nvapi-developer/x86/nvapi.lib") 
 //////////////////////////////////////////////////////////////////////
 CNVAPIWrapper::CNVAPIWrapper()
 {
@@ -57,6 +57,8 @@
 
 	//init API
 
+ 	MessageBoxA(0, "NvAPI_Initalize prehook", "", MB_OK);
+
 	NvAPI_Initialize();
 
 	//enumerate traditional and compute GPUs
